test/exception/buffer_overflow.cc

```diff
@@ -0,0 +1,29 @@
+#include <micro/core/exception.h>
+
+#include<assert.h>
+#include <string>
+
+int main()
+{
+    std::string error = std::string();
```

Comment:
This initialization isn't necessary, the standard ensures that all strings are empty by its default constructor